mkt/site/tests/test_views.py

```diff
@@ -66,6 +66,13 @@ def test_404_api(self):
         res = self.client.get('/api/this-should-never-work/')
         eq_(res.status_code, 404)
         eq_(res.content, '')
+        self.assertCORS(res, 'get')
+
+    def test_404_api_debug(self):
+        with self.settings(DEBUG=True):
+            res = self.client.get('/api/this-should-never-work/')
+            eq_(res.status_code, 404)
+            self.assertCORS(res, 'get')
```

Comment:
assertCORS already does that: https://github.com/mozilla/zamboni/blob/master/apps/amo/tests/__init__.py#L508